### ISSUE TITLE
Signature/proof formats are undocumented

### ISSUE THREAD

**Issue by andrewwhitehead** [w3c-ccg/ldp-bbs2020#61]
Unless I've missed it, neither this specification nor the BBS+ signatures spec defines the encoding for a BBS+ signature or for a PoK of a signature. The signature is fairly simple, although the compression and endianness need to be explicit. The proof encoding is rather more complicated.

**Comment by andrewwhitehead** [comment]
Yup, related issue: https://github.com/decentralized-identity/bbs-signature/issues/2

**Comment by OR13** [comment]
This is still critical to fix, the document needs to be pointed to the latest IETF drafts.

**Comment by Wind4Greg** [comment]
Updated document contains full definitions of signature and proof formats along with test vectors.
